src/automatic.ts

```diff
@@ -228,44 +254,40 @@ export class AutomaticMode {
           `in ${spec.browser.name} from ${url}. Retrying.`);
     }
 
-    if (millis === undefined) {
+    if (numPending > 0) {
       console.log();
       throw new Error(
           `\n\nFailed ${maxAttempts}/${maxAttempts} times ` +
           `to get a measurement ` +
```

Comment:
In the case of multiple measurements, it'd be useful to know which measurement(s) are missing

Reply:
Done.